### PR TITLE
Update intro diagram to show flow of visas instead of flow of requests

### PR DESCRIPTION
**Pull request by mbarkley**

### CODE REVIEW

File: AAI/README.md [hunk 1]
```diff
@@ -97,10 +97,10 @@ component "<b>Visa Issuer N</b>\nservice" as IssuerN
...
 component "<b>Passport Clearinghouse</b>\nservice" as ClearingHouse #9E7BB5
 
-Issuer1 <-- Broker : Fetch Visas
-Issuer2 <-- Broker : Fetch Visas
-IssuerN <-- Broker : Fetch Visas
-Broker <-- ClearingHouse : Request User Visas w/ Access Token
+Issuer1 --> Broker : Provide Visas
+Issuer2 --> Broker : Provide Visas
+IssuerN --> Broker : Provide Visas
+Broker --> ClearingHouse : Provide Visas
```
Comment (andrewpatto) [inline AAI/README.md:103]: "Provide Passport with Visas" ?

Comment (andrewpatto) [reply AAI/README.md:103]: For the broker to clearing house..

**Comment by TomConner** [comment]
https://ga4gh.github.io/data-security/intro-diagram-update/aai-introduction

File: AAI/README.md (hunk 1 above)
Comment (TomConner) [reply AAI/README.md:103]: "Provide Passport and Visas" maybe? Either way.

**Review by TomConner — APPROVED** [review]

**Pull request merged**